docs/tutorial/GetStarted.md

```diff
@@ -0,0 +1,392 @@
+
+#### Change port mapping to be dynamic
+
+If we want to get a port number dynamically from available ports, Marathon provides a way to achieve this. We access the available ports using the environment variable `$PORT0`, `$PORT1`, `$PORT2` and so on. This will be explained clearly in a later section. But for now, just change your Python snippet to read the port from an environment variable as below :
+
+```python
+# Gets the port number from $PORT0 environment variable
+PORT_NUMBER = int(os.environ['PORT0'])
+```
+
+Once you do this, you can browse your server after executing `PORT0=8000 python3 helloworld.py`
+
+### Step 2 : Creating a Docker container
+Creating a Docker container is essential to distribute _this_ service. It runs completely isolated from the host environment by default, only accessing host files and ports if configured to do so. To continue reading, you need to be familiar with Docker; we recommend this [get-started](https://docs.docker.com/get-started/) guide. You should have logged in to your Docker account in your terminal using `docker login`.
+
+_Note: Giving a Docker image is optional and we can have other ways to execute the binary (E.g.: The package `cassandra` doesn't use a Docker image to install the binary.)._
+
+We create a Docker file (named `Dockerfile`) in the `time-server-service` directory created earlier. The `Dockerfile` should look like this:
+
+```
+# Use an official Python runtime as a base image
+FROM python:3
+
+# Set the working directory to the package directory
+WORKDIR /package
+
+# Copy the current directory contents into the container at /package
+ADD . /package
+
+# Run helloworld.py when the container launches. -u flag makes sure the output is not buffered
+CMD ["python3", "-u", "helloworld.py"]
+```
+Read through the comments to understand what each line in the `Dockerfile` does.
+
+
+#### Build the container
+
+*Throughout the rest of this guide, we refer to `docker-user-name` as your Docker user name where you access your Docker images from a registry such as Docker Hub. You are expected to replace the keyword `docker-user-name` with your Docker user name in all commands and files*
+
+Now that we have everything ready, we can build our container. Here’s what `ls` should show:
+```
+$ ls
+Dockerfile		helloworld.py
+```
+Now run the build command. This creates a Docker image which we’re going to tag using `-t` so it has a friendly name.
+
+`docker build -t docker-user-name/time-server:part1 .`
+
+Where is your built image? It's in your machine's local Docker daemon:
+
+```
+$ docker images
+
+REPOSITORY                        TAG         IMAGE ID
+docker-user-name/time-server      part1       42somsoc147
+```
+
+
+#### Test your container
+When you execute `docker images`, you should be able to see the your image in the displayed list. In order to make sure the image is working as expected, you can run the container by issuing the below command :
```

Comment:
Remove unnecessary `In order to` and reword as `To make sure the image is working as expected...`